### ISSUE TITLE
get_type() does not return "SELECT" if a query uses CTEs

### ISSUE THREAD

**Issue by andrew-tipton-cgg** [andialbrecht/sqlparse#227]
Calling `get_type()` on a SQL query that uses a CTE (`WITH ... SELECT ...`) returns `UNKNOWN` instead of `SELECT`.  Test case:

```
import sqlparse

select_stmt = sqlparse.parse('SELECT 1, 2, 3 FROM foo;')[0]
assert select_stmt.get_type() == 'SELECT'
with_stmt = sqlparse.parse('WITH foo AS (SELECT 1, 2, 3) SELECT * FROM foo;')[0]
assert with_stmt.get_type() == 'SELECT'  # returns 'UNKNOWN'
```

**Comment by andrew-tipton-cgg** [comment]
(I'm using sqlparse version *0.1.18*, the most recent version.)

**Comment by andialbrecht** [comment]
Fixed in 8f39d3367e8878e5f9a09b0166e3f0f73626e67d
